README.rst

```diff
@@ -495,6 +499,13 @@ Choose from the list of available fixers:
                         An empty line feed should
                         precede a return statement.
 
+* **scalar_cast** [symfony]
```

Comment:
And we would change that `integer` into `int`.
So, as both form could be considered as correct, the choose of form is purely CS matter. Therefore, `short_scalar_cast` provides non-uncertain name